api/serverpb/server.proto

```diff
@@ -181,7 +182,18 @@ message GetSettingsResponse {
 }
 
 message ChangeSettingsRequest {
-  bool enable_telemetry = 1;
+  option (grpc.gateway.protoc_gen_swagger.options.openapiv2_schema) = {
+    json_schema: {
+      title: "Super puper message title!"
+      description: "Awesome message description"
+    }
+  };
+  bool enable_telemetry = 1 [
+    (grpc.gateway.protoc_gen_swagger.options.openapiv2_field) = {
+      title: "Super puper field title!"
+      description: "Awesome field description"
+    }
+  ];
```

Comment:
Field title and descriptions

p.s. Descriptions already present in our docs

Reply:
I vaguely remember that both title and description can be set without annotation if you end sentences with dots

Reply:
The whole comment visualized as a description, but I think titles are useless for fileds.
```
  // A number of full days for Prometheus and QAN data retention. Should have a suffix in JSON: 2592000s, 43200m, 720h.
  google.protobuf.Duration data_retention = 4;
  ```
  
![Screen Shot 2021-02-11 at 12 43 59](https://user-images.githubusercontent.com/21174514/107626700-dcb14080-6c66-11eb-82ab-0d6b387fde9b.png)